recipes/qcs-sdk-python/meta.yaml

```
@@ -0,0 +1,126 @@
+{% set name = "qcs-sdk-python" %}
+{% set version = "0.17.8" %}
+
+package:
+  name: qcs-sdk-python-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/rigetti/qcs-sdk-rust/archive/refs/tags/python/v{{ version }}.tar.gz
+  sha256: 0151a649dd670cdc85c7659bc6c82066f15259bf9b9cc1151e72fea9dc58ddd2
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

You need to add the stdlib everywhere where there is a compiler.
```suggestion
    - {{ stdlib('c') }}
    - {{ compiler('c') }}
```

---

Would this conflict with using `sysroot >=2.28`? The other difficult project I am working on seem to require 2.28

---

No, one point why we introduced this is to easily select the `2.28` sysroot. No need to say `sysroot` nowadays, only bump `c_stdlib_version` to `2.28` in `cbc.yaml`.

---

Cool, thx. I am having some difficulties with building sbcl, the tests fail with:
```
The following package could not be installed
└─ sbcl 2.4.4.* h95efde1_0 is not installable because it requires
   └─ __glibc >=2.28,<3.0.a0 , which is missing on the system.
```
I was wanting to figure it out before making the PR, but maybe I should PR and you could look at it? it builds and runs the tests if I test while building - odd thing is that I think I had a configuration where it worked